### ISSUE TITLE
[Feature Request] Add option/page to hide boosts when viewing a profile directly

### ISSUE THREAD

**Issue by killerbee13** [tuskyapp/Tusky#1607]
Often when I go to a profile, I want to see the toots which that person has actually written, but there is no way in the current UI to do that. The only pages available are "posts", "with replies", "pinned", and "media".

I do not want a setting to hide boosts in general, I want this to be toggleable for an individual visit to a profile.

* * * *
- Tusky Version: 9.1
- Android Version: 9
- Android Device: moto x4
- Mastodon instance (if applicable): N/A

- [ x ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by Ghryphen** [comment]
Agreed, when I go to a user's profile I am looking to learn more about them through their original posts. Would like to see Boosts as their own tab.

**Comment by barometz** [comment]
To add motivation to this: I follow a fair number of people with boosts disabled. When someone boosts a lot (or much more than they toot), I want to see what following them but not their boosts would do to my timeline.

**Comment by yelworc** [comment]
I know +1 comments are dumb, but @barometz and @Ghryphen have already perfectly articulated the two reasons why I would appreciate this option, so… I don't really have anything to add but +1 😆

(also curious if this needs protocol support and we should pester the Mastodon folks instead)
